Adds Support For SCTs in OCSP Response

droptableuser opened the pull request:
parser to the ocsp response was added to nassl/ocsp_response.py to
enable the retrieval of SCTs from the ocsp response.

[ticket: #4]

craSH commented:
This open pull request is making it so I can't initiate a request for another one.. I made two different commits just now, one is related to this (update the fetch script), but the other (a3df442b8be5b7c169938d5a9ba63d9fd8f16779) is just a fix for the shebang lines on the scripts, which was breaking the build on OSX with custom versions of Python installed. That commit just unifies all the shebang lines to #!/usr/bin/env python2.7

Could you close this request so I can open that one, or accept this?

Also wow; just saw that this made it in to homebrew literally a day ago, so this may be far less relevant now. Good work to whoever fought that one, they always pushed back on me for trying to get it in there due to the odd dependencies/binary requirements.

nabla-c0d3 commented:
Sounds good, I will close this request as sslyze is now on Homebrew (at last) :)

droptableuser committed:
Adds Support For SCTs in OCSP Response

parser to the ocsp response was added to nassl/ocsp_response.py to
enable the retrieval of SCTs from the ocsp response.

[ticket: #4]

nabla-c0d3 commented:
Thanks! Does it require a specific version of OpenSSL or did you use the fork?
